internal/provider/provider.go

```diff
@@ -128,7 +128,8 @@ func New() *schema.Provider {
 			"boundary_worker":                                   resourceWorker(),
 		},
 		DataSourcesMap: map[string]*schema.Resource{
-			"boundary_scope": dataSourceScope(),
+			"boundary_scope":                   dataSourceScope(),
+			"boundary_credential_store_static": dataSourceCredentialStoreStatic(),
```

Comment:
I'm not sure I follow how to break down these resources. What's the reason for creating a specific `boundary_credential_store_static` data source as opposed to a `boundary_credential_store` one?